deploy_slave.sh

```diff
@@ -78,8 +78,7 @@ getent group piwheels || groupadd piwheels
 getent passwd piwheels || useradd -g piwheels -m -s /bin/bash piwheels
 passwd -d piwheels
 
-curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
-cat .cargo/env >> /home/piwheels/.bashrc
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | runuser -- - piwheels -s -- -y
```

Comment:
```suggestion
curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | runuser -- - piwheels -s -- -y --profile minimal
```
The `minimal` profile for rustup seems to be more suitable, because it doesn't include the docs, clippy (linter), or rustfmt (formatter). If a pip wheel depends on the rust docs (not rust-doc, which comes with rustc), we can still use minimal but add it with -c

Reply:
I would have suggested the exact same in a dedicated PR, to keep changes here related to the subject. However, if @bennuttall agrees we can include it here.